resources/user-agents/crawlers/search-engines/search-engines.json

```diff
@@ -1185,6 +1185,48 @@
             "Browser": "WebCorp",
             "Browser_Maker": "Research and Development Unit for English Studies, Birmingham City University"
           }
+        },
+        {
+          "match": "Mozilla/5.0 (compatible; coccocbot-web/*",
```

Comment:
Please move this pattern to the other `coccoc` pattern in this file (around line 920) and update the browser maker.